src/Timeline.js

```diff
@@ -187,6 +213,22 @@ export default class Timeline extends Axis {
       .call(attrize, this._handleConfig)
       .attr("height", timelineHeight + this._handleSize);
 
+    if (this._buttonBehaviorCurrent === "buttons") {
+
+      const yTransform = this._height / 2 - this._buttonHeight / 2;
+
+      this._brushGroup.selectAll(".selection")
+      .call(attrize, this._selectionConfig)
+      .attr("height", timelineHeight)
+      .attr("y", yTransform);
```

Comment:
you're repeating the `attrize` and `height` setting from above, just move the `y` attribute to the lines above outside of this IF statement (maybe make the `yTransform` variable a ternary that checks the buttonbehvaior?)

Reply:
Move these lines outside of the IF statement and make them inline ternaries (so we're not using `selectAll` twice in a row). Example:

```js
const yTransform = this._buttonBehaviorCurrent === "buttons" ? this._height / 2 - this._buttonHeight / 2 : 0;

this._brushGroup.selectAll(".selection")
  .call(attrize, this._selectionConfig)
  .attr("height", timelineHeight)
  .attr("y", yTransform)
```

Reply:
When I converted `yTransform` to ternary operator, I had problems with `y` in timeline with ticks. I proposed other way for to improve this.